src/relay/ir/expr.cc

```diff
@@ -27,6 +27,26 @@
 
 namespace tvm {
 
+GlobalVar WithFields(GlobalVar global_var, Optional<String> opt_name_hint, Optional<Type> opt_type,
```

Comment:
Could you add some tests for `GlobalVar` and `Constant` variants of `WithFields`?

Reply:
Done.